perfmetrics/scripts/ml_tests/pytorch/dino/setup_container.sh

```diff
@@ -1,6 +1,7 @@
 #!/bin/bash
 
-wget -O go_tar.tar.gz https://go.dev/dl/go1.20.4.linux-amd64.tar.gz
+# Install golang
+wget -O go_tar.tar.gz https://go.dev/dl/go1.20.5.linux-amd64.tar.gz
```

Comment:
Also use CGO_ENABLED in go build command in this file

Reply:
also check in perfmetrics/scripts/ml_tests/tf/resnet/setup_scripts/setup_container.sh

Reply:
Thanks done.